XpringKit/XRP/XRPLedgerError.swift

```diff
@@ -9,6 +9,12 @@ public enum XRPLedgerError: Equatable, Error {
   /// An invalid address was given. A more descriptive string is provided as an associated value.
   case invalidInputs(String)
 
+  /// An invalid drops amount was given.  A more descriptive string is provided as an associeted value.
```

Comment:
I'd bundle these into invalid inputs unless you really think they're needed.

Reply:
nope.  was thrown off by the 'an invalid address was provided' comment but I just updated that to make it more generic.